src/apps/explorer/components/SummaryCardsWidget/SummaryCards.tsx

```diff
@@ -142,7 +143,7 @@ export function SummaryCards({ summaryData, children }: SummaryCardsProps): JSX.
           <CardContent
             variant={rowsByCard}
             label1="24h Fees"
-            value1={`$${dailyFees?.now.toFixed(2)}`}
+            value1={`$${numberFormatter(dailyFees?.now || 0)}`}
```

Comment:
Any particular reason for using this rather than `formatSmart`?

Reply:
is formatSmart displaying K for thousands or M for millions? 

<img width="120" alt="image" src="https://user-images.githubusercontent.com/3328006/166295613-5f51f8ce-ff3e-4b3c-8edc-ffa6ce292b40.png">

Reply:
Hmmm, not sure, but I see your point.